ui/app/templates/vault.hbs

```diff
@@ -4,32 +4,23 @@
 ~}}
 
 {{outlet}}
-<footer class="footer has-text-grey has-text-centered">
-  <span class="is-inline-block">
-    <Icon @name="hashicorp" aria-hidden="true" class="has-text-grey-light" />
-    &copy;
-    {{date-format (now) "yyyy"}}
-    HashiCorp
-  </span>
-  <span>
-    <ExternalLink @href={{changelog-url-for this.auth.activeCluster.leaderNode.version}} class="link has-text-grey">
-      Vault
-      {{this.auth.activeCluster.leaderNode.version}}
-    </ExternalLink>
-  </span>
+
+<Hds::AppFooter as |AF|>
+  <AF.Link @href={{changelog-url-for this.auth.activeCluster.leaderNode.version}}>
+    Vault
+    {{this.auth.activeCluster.leaderNode.version}}
+  </AF.Link>
   {{#if (is-version "OSS")}}
-    <span>
-      <ExternalLink @href="https://hashicorp.com/products/vault/trial?source=vaultui" class="link has-text-grey">
-        Upgrade to Vault Enterprise
-      </ExternalLink>
-    </span>
+    <AF.Link @href="https://hashicorp.com/products/vault/trial?source=vaultui">
+      Upgrade to Vault Enterprise
+    </AF.Link>
   {{/if}}
-  <span>
-    <DocLink @path="/vault" class="has-text-grey">
-      Documentation
-    </DocLink>
-  </span>
-</footer>
+  <AF.Link @href="https://developer.hashicorp.com/vault">
```

Comment:
note: can't use `DocLink` anymore so had to hardcode this link it for now

Reply:
we have a template helper you can use:

here's an example of where else it's implemented
https://github.com/hashicorp/vault/blob/94fe0f4f0dd89e71530fddb08d958cbcbc60a047/ui/lib/sync/addon/components/secrets/landing-cta.hbs#L13